docs/reference/samples/observability.output-splunk.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: instance
```

Comment:
Just a question, should we move away from suggesting the name `instance` for CLFs?

Reply:
Maybe?   I'll change it anyways, it was an old file I had sitting around.   Good catch.